src/util/datablock/datablock.c

```diff
@@ -180,3 +167,11 @@ void DataBlock_Free(DataBlock *dataBlock) {
 	assert(pthread_mutex_destroy(&dataBlock->mutex) == 0);
 	rm_free(dataBlock);
 }
+
+/* --------- DataBlock Item API implementation --------*/
+
+inline DataBlockItemHeader *DataBlock_GetItemHeader(const DataBlock *dataBlock, uint64_t idx) {
```

Comment:
Also I believe this function should be an internal function, not exposed via the header.

Reply:
used in `oo_datablock`